CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
 
 ##### Enhancements
 
+* Add ability to specify the `validation-dir` during `lint`.    
```

Comment:
super nit: this has 4 spaces at the end instead of 2

Reply:
fixed it in a post commit.